--- test/net/http/test_https.rb.orig    2021-01-03 00:03:24.000000000 +0900
+++ test/net/http/test_https.rb 2021-01-03 00:03:26.000000000 +0900
@@ -45,7 +45,7 @@
       assert_equal($test_net_http_data, res.body)
     }
     # TODO: OpenSSL 1.1.1h seems to yield only SERVER_CERT; need to check the incompatibility
-    certs.zip([SERVER_CERT, CA_CERT]) do |actual, expected|
+    certs.zip([CA_CERT, SERVER_CERT]) do |actual, expected|
       assert_equal(expected.to_der, actual.to_der)
     end
   rescue SystemCallError
